<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wordle</class>
 <widget class="QDialog" name="Wordle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="Qbutton">
      <property name="text">
       <string>Q</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Wbutton">
      <property name="text">
       <string>W</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Ebutton">
      <property name="text">
       <string>E</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Rbutton">
      <property name="text">
       <string>R</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Tbutton">
      <property name="text">
       <string>T</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Ybutton">
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Ubutton">
      <property name="text">
       <string>U</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Ibutton">
      <property name="text">
       <string>I</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Obutton">
      <property name="text">
       <string>O</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Pbutton">
      <property name="text">
       <string>P</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="Abutton">
      <property name="text">
       <string>A</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Sbutton">
      <property name="text">
       <string>S</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Dbutton">
      <property name="text">
       <string>D</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Fbutton">
      <property name="text">
       <string>F</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Gbutton">
      <property name="text">
       <string>G</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Hbutton">
      <property name="text">
       <string>H</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Jbutton">
      <property name="text">
       <string>J</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Kbutton">
      <property name="text">
       <string>K</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Lbutton">
      <property name="text">
       <string>L</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>650</y>
     <width>461</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="enterButton">
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Zbutton">
      <property name="text">
       <string>Z</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Xbutton">
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Cbutton">
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Vbutton">
      <property name="text">
       <string>V</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Bbutton">
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Nbutton">
      <property name="text">
       <string>N</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Mbutton">
      <property name="text">
       <string>M</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="backButton">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>421</width>
     <height>452</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QTextEdit" name="text0_2">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QTextEdit" name="text0_3">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTextEdit" name="text0_0">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QTextEdit" name="text3_0">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTextEdit" name="text1_0">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QTextEdit" name="text4_0">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTextEdit" name="text0_1">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTextEdit" name="text2_0">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QTextEdit" name="text0_4">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QTextEdit" name="text5_0">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QTextEdit" name="text1_1">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QTextEdit" name="text1_2">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QTextEdit" name="text1_3">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QTextEdit" name="text1_4">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QTextEdit" name="text2_1">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QTextEdit" name="text2_2">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QTextEdit" name="text2_3">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QTextEdit" name="text2_4">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTextEdit" name="text3_1">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QTextEdit" name="text3_2">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QTextEdit" name="text3_3">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QTextEdit" name="text3_4">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QTextEdit" name="text4_1">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QTextEdit" name="text4_2">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QTextEdit" name="text4_3">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QTextEdit" name="text4_4">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QTextEdit" name="text5_1">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QTextEdit" name="text5_2">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QTextEdit" name="text5_3">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QTextEdit" name="text5_4">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>25</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>You win! Here is Wordle</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
